### ISSUE TITLE
AWS::EC2::VPCEndpoint Tags not allowed

### ISSUE THREAD

**Issue by ctp-consulting** [aws-cloudformation/cfn-lint#3894]
### CloudFormation Lint Version

1.22.2

### What operating system are you using?

Windows

### Describe the bug

The following error appears when Tags are specified for a VPC Endpoint resource.
[cfn-lint] E3002: Additional properties are not allowed ('Tags' was unexpected)

cfn-lint is used from VS Code's CloudFormation Linter extension

### Expected behavior

As shown below, it is configurable in the Cloudformation configuration and should not cause errors in cfn-lint.
https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpcendpoint.html

### Reproduction template

sample code
```
AWSTemplateFormatVersion: "2010-09-09"
Transform: AWS::Serverless-2016-10-31
Description: This is the Hoge VPC network backend stack.

Resources:
  HogeEC2VPCEndpointS3:
    Type: "AWS::EC2::VPCEndpoint"
    Properties:
      VpcEndpointType: "Gateway"
      VpcId: !Ref HogePublicVPC
      ServiceName: !Sub "com.amazonaws.${AWS::Region}.s3"
      RouteTableIds:
        - !Ref HogePublicRouteTable
        - !Ref HogePrivateSubnetRouteTable
      PrivateDnsEnabled: false
      Tags:
        - Key: SystemName
          Value: Hoge
```

**Comment by kddejong** [comment]
I cannot replicate this issue with version `1.22.3`.  You just need to do an upgrade.

Tagging support is [there](https://github.com/aws-cloudformation/cfn-lint/blob/f5084b0c87696260c4ccfcc77105a203a4679143/src/cfnlint/data/schemas/providers/us_east_1/aws-ec2-vpcendpoint.json#L194-L200)

Looks like support was recently added as this is the definition of [tags in v1.22.2](https://github.com/aws-cloudformation/cfn-lint/blob/700563cda182416a6c99af4bcaa96077c2412465/src/cfnlint/data/schemas/providers/us_east_1/aws-ec2-vpcendpoint.json#L123-L128)

You can also run `cfn-lint --update-specs` on v1.22.2
